README.md

```diff
@@ -57,7 +57,24 @@ want to adjust the config file if you want to connect to the lab's server
 or run multiple agents. You will also need to adjust the config if you
 want to play with the keyboard against your agent.
 
-### 4. (Optional) Start a local server for testing
+### 4. Setup your agent file
+
+Copy `common/agents/agent.py.template` to `common/agents/agent.py`. You can use your graphical interface or one of the following commands:
```

Comment:
Do you want to say something like: "if you don't have any existing agent.py files, copy common/agents/agent.py.template to common/agents/agent.py."?